src/olympia/blocklist/views.py

```diff
@@ -82,17 +87,26 @@ def clear_blocklist(*args, **kw):
                                    dispatch_uid='delete_%s' % m)
 
 
-def get_items(apiver, app, appver=None):
+def get_items(apiver=None, app=None, appver=None):
     # Collapse multiple blocklist items (different version ranges) into one
     # item and collapse each item's apps.
+
+    app_query = (Q(app__isnull=True) |
```

Comment:
`if app:`/`else:` instead of setting this and then overwriting it `if app` below ?